### ISSUE TITLE
Colour contrast ratio can be selected but not copied

### ISSUE THREAD

**Issue by nilamf** [WebAccessibilityTools/CCA#32]
There is an inconsistency when copying colour details. While I can select and copy (with tthe right-click context menu) the colour name and hex code, the contrast ratio cannot be copied even though the text is selectable.

Version: v0.1.8 for MAC OS.
Test device: MacBook Pro macOS 15.7.5 (24G624)

<img width="512" height="551" alt="Image" src="https://github.com/user-attachments/assets/98cced3d-aa62-43df-9469-95a0c0f3e5b8" />

**Comment by ferllings** [comment]
Fixed with a0560560f7bd9675d947f4597efe2c36c53d70c3
